### PR TITLE
Make MeetYourHostSection component responsive and dynamic

### PR DESCRIPTION
**Pull request by grabinskij**
This PR makes the meetYourHostSection responsive and dynamic. Also the broken pictures in place.json for meetYourHostSection were replaced with new ones.

It will be easier to review after Feature/responsive gallery #348 will be merged since the components from #348  PR will have no bad impact to the current component.

meetHost

### CODE REVIEW

File: src/pages/ProductPage.jsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ const ProductPage = () => {
   const [error, setError] = useState(null);
 
   const isInitializedRef = useRef(false)
-
+console.log('place', place)
```
Comment (orYoffe) [inline src/pages/ProductPage.jsx:44]: do we need this here?

**Comment by orYoffe** [comment]
I can't see the area.. is there a way to validate the PR otherwise?
Screenshot 2024-12-03 at 18 56 49

**Comment by orYoffe** [comment]
> by removing other components in the browser

I'd say "by removing other components in the browser" is not really great.. anyway you can make it actually work? 😄

File: src/pages/ProductPage.jsx (hunk 1 above)
Comment (grabinskij) [reply src/pages/ProductPage.jsx]: Thanks you noticed it. I have it deleted.

**Comment by grabinskij** [comment]
> > by removing other components in the browser
> 
> I'd say "by removing other components in the browser" is not really great.. anyway you can make it actually work? 😄

I have deleted all components exept Header, otherwise it couldnot be posible for me separate the component and work with it.

**Review by 006080 — APPROVED** [review]

**Pull request merged**